examples/service-catalog/service-catalog.yaml

```diff
@@ -0,0 +1,370 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: service-catalog
+objects:
+
+- kind: ClusterRoleBinding
```

Comment:
off the top of my head, this is missing the necessary permissions for SC SA to create Secrets and PodPresets in any namespace. Needed for binding to work correctly.